src/test/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsControllerTests.java

```diff
@@ -45,742 +45,892 @@
-          .contentType(MediaType.APPLICATION_JSON)
-                      .characterEncoding("utf-8")
-                      .content(requestBody)
-                      .with(csrf()))
-              .andExpect(status().is(400)).andReturn();
-  
-      // assert
-      String responseString = response.getResponse().getContentAsString();
-      String expectedString = "{\"message\":\"You need more money!\",\"type\":\"NotEnoughMoneyException\"}";
-      Map<String, Object> expectedJson = mapper.readValue(expectedString, Map.class);
-      Map<String, Object> jsonResponse = responseToJson(response);
-      assertEquals(expectedJson, jsonResponse);
-  }
-  
-  @WithMockUser(roles = { "USER" })
-  @Test
-  public void test_SellCow_commons_exists_no_cow_to_sell() throws Exception {
-  
-      // arrange
-  
-      UserCommons origUserCommons = UserCommons
-      .builder()
-      .id(1L)
-      .userId(1L)
-      .commonsId(1L)
-      .totalWealth(300)
-      .numOfCows(0)
-      .cowHealth(100)
-      .build();
-  
-      Commons testCommons = Commons
-      .builder()
-      .name("test commons")
-      .cowPrice(10)
-      .milkPrice(2)
-      .startingBalance(300)
-      .startingDate(LocalDateTime.now())
-      .build();
-  
-      UserCommons userCommonsToSend = UserCommons
-      .builder()
-      .id(1L)
-      .userId(1L)
-      .commonsId(1L)
-      .totalWealth(300)
-      .numOfCows(0)
-      .cowHealth(100)
-      .build();
-  
-      UserCommons correctuserCommons = UserCommons
-      .builder()
-      .id(1L)
-      .userId(1L)
-      .commonsId(1L)
-      .totalWealth(300)
-      .numOfCows(0)
-      .cowHealth(100)
-      .build();
-  
-      String requestBody = mapper.writeValueAsString(userCommonsToSend);
-      String expectedReturn = mapper.writeValueAsString(correctuserCommons);
-  
-      when(userCommonsRepository.findByCommonsIdAndUserId(eq(1L), eq(1L))).thenReturn(Optional.of(origUserCommons));
-      when(commonsRepository.findById(eq(1L))).thenReturn(Optional.of(testCommons));
-  
-      // act
-      MvcResult response = mockMvc.perform(put("/api/usercommons/sell?commonsId=1")
-          .contentType(MediaType.APPLICATION_JSON)
-                      .characterEncoding("utf-8")
-                      .content(requestBody)
-                      .with(csrf())).andExpect(status().is(400)).andReturn();
-
-
-      // assert
-      String responseString = response.getResponse().getContentAsString();
-      String expectedString = "{\"message\":\"You have no cows to sell!\",\"type\":\"NoCowsException\"}";
-      Map<String, Object> expectedJson = mapper.readValue(expectedString, Map.class);
-      Map<String, Object> jsonResponse = responseToJson(response);
-      assertEquals(expectedJson, jsonResponse);
+    @MockBean
+    UserCommonsRepository userCommonsRepository;
+
+    @MockBean
+    UserRepository userRepository;
+
+    @MockBean
+    CommonsRepository commonsRepository;
+
```

Comment:
Are there actually any changes in this file, or is it just "white space" changes?